openreview/conference/builder.py

```diff
@@ -493,13 +493,14 @@ def open_revise_reviews(self, name = 'Revision', start_date = None, due_date = N
     def close_revise_submissions(self, name):
         return self.__expire_invitations(name)
 
-    def set_program_chairs(self, emails):
+    def set_program_chairs(self, emails = [], enable_reviewer_reassignment = True):
         self.__create_group(self.get_program_chairs_id(), self.id, emails)
         ## Give program chairs admin permissions
         self.__create_group(self.id, '~Super_User1', [self.get_program_chairs_id()])
+        self.enable_reviewer_reassignment = enable_reviewer_reassignment
```

Comment:
are we using the same property to enable reassignment in both consoles? in that case we prefer to set the property using the builder directly instead of setting it through set_program_chairs or set_area_chairs